pkg/envtest/server.go

```diff
@@ -113,6 +116,10 @@ type Environment struct {
 	// values are merged.
 	CRDDirectoryPaths []string
 
+	// AbsBinaryDirectory is the path where the binaries required for the envtest are
+	// locate in the environment
+	AbsBinaryDirectory string
```

Comment:
This name is a bit unclear.  Can we make this `TestBinariesDirectory` or `BinaryAssetsDirectory` or something?

Reply:
done